src/test/regress/citus_tests/config.py

```diff
@@ -211,34 +211,14 @@ def __init__(self, arguments):
         super().__init__(arguments)
         self.all_null_dist_key = True
         self.skip_tests += [
-            # i) Skip the following tests because they require SQL support beyond
-            #    router planner / supporting more DDL command types.
-            #
-            # group 1
-            "dropped_columns_create_load",
-            "dropped_columns_1",
-            # group 2
-            "distributed_planning_create_load",
-            "distributed_planning",
-            # group 4
-            "views_create",
-            "views",
-            # group 5
-            "intermediate_result_pruning_create",
-            "intermediate_result_pruning_queries_1",
-            "intermediate_result_pruning_queries_2",
-            # group 6
-            "local_dist_join_load",
-            "local_dist_join",
-            "arbitrary_configs_recurring_outer_join",
-            # group 7
-            "sequences_create",
-            "sequences",
-            # group 8
+            # One of the distributed functions created in "function_create"
+            # requires setting a distribution column, which cannot be the
+            # case with single shard tables.
             "function_create",
             "functions",
-            #
-            # ii) Skip the following test as it requires support for create_distributed_function.
+            # In "nested_execution", one of the tests that query
```

Comment:
> -- nested execution is allowed outside of an aggregate
-- note that this behaviour is different if distributed has only 1 shard
-- however, this test always uses 4 shards
SELECT dist_query_single_shard(count(*)::int) FROM distributed;

Reply:
what happens when you enable this test? It does seem to query a table? maybe change the query to `SELECT dist_query_single_shard(count(*)::int) == X ROM distributed;`, where `X` is what changes depending on the shard count?

Reply:
We've decided to not enable this tests for single shard tables because there are lots of tests in this test file that fail when the table has one shard.